packages/nimble-components/src/table/tests/table.pageobject.ts

```diff
@@ -25,7 +26,25 @@ export class TablePageObject<T extends TableRecord> {
             );
         }
 
-        return headers.item(columnIndex).textContent?.trim() ?? '';
+        return this.getHeaderContentElement(headers[columnIndex]!);
+    }
+
+    public getRenderedHeaderContent(
+        columnIndex: number
+    ): Node | null | undefined {
+        const headers = this.tableElement.shadowRoot!.querySelectorAll<TableHeader>(
```

Comment:
Can you just call `getHeaderContent` and then call `.firstChild` on the result? The implementations look pretty much the same.

Reply:
+1 for this suggestion. It's also unclear to me the difference between `getRenderedHeaderContent` and `getHeaderContent`, so if we could just have one, I think that would be better.